### PR TITLE
LG-3776 Log vendor resolution transaction IDs

### PR DESCRIPTION
**Pull request by stevegsa**

### CODE REVIEW

File: lib/lexisnexis/proofer.rb [hunk 1]
```diff
@@ -2,6 +2,7 @@ module LexisNexis
   class Proofer < Proofer::Base
     def proof_applicant(applicant, result)
       response = send_verification_request(applicant)
+      result.transaction_id = response.conversation_id
```
Comment (zachmargolis) [inline lib/lexisnexis/proofer.rb:5]: what type of object is `result`? does it already have this method?

File: spec/support/shared_examples/proofer.rb [hunk 2]
```diff
@@ -1,11 +1,13 @@
 shared_examples 'a proofer' do
   let(:verification_status) { 'passed' }
+  let(:conversation_id) { 'foo' }
   let(:verification_errors) { {} }
   let(:result) { Proofer::Result.new }
 
   before do
     response = instance_double(LexisNexis::Response)
     allow(response).to receive(:verification_status).and_return(verification_status)
+    allow(response).to receive(:conversation_id).and_return(conversation_id)
```
Comment (zachmargolis) [inline spec/support/shared_examples/proofer.rb:10]: can we add a part of the spec that reads back the transaction ID we set from this?

File: lib/lexisnexis/proofer.rb (hunk 1 above)
Comment (stevegsa) [reply lib/lexisnexis/proofer.rb:5]: It's a `ProoferResult`.  I had to add the `transaction_id` to it.

Comment (stevegsa) [reply lib/lexisnexis/proofer.rb:5]: `Proofer::Result` is in here: https://github.com/18F/identity-proofer-gem/pull/48

**Review by zachmargolis — APPROVED** [review]
LGTM, let's update the VERSION in this constant gem too

**Pull request merged**